contributing.rst

```diff
@@ -1576,15 +1576,46 @@ Video files should be put in the :file:`/vid/` directory in the source, and they
 
 The length of the videos must be less than a minute.
 
-There is no ``video`` directive to add a video, so to add a video in a document, you can do the following:
+We have a custom ``video`` directive to add a video.
+
```

Comment:
Put a simple example (no changes to default behavior) here, before the customization details.